Quasardb/QdbCluster.cs

```diff
@@ -77,6 +77,18 @@ public QdbCluster(string uri, string clusterPublicKey, string userName, string u
             _factory = new QdbEntryFactory(_handle);
         }
 
+        /// <summary>
+        /// Gets the last API error description.
+        /// <returns>A message describing the error occurred during the last operation.</returns>
+        /// </summary>
+        public string GetLastError()
+        {
+            qdb_error error;
+            qdb_sized_string message;
+            qdb_api.qdb_get_last_error(out error, out message);
```

Comment:
You can pass NULL instead of error if you don't want it (if .NET allows it).

Reply:
Right.

Reply:
But I'm not sure how to do it :/.

Reply:
Let's keep it like that ^^